.travis.yml

```diff
@@ -11,14 +15,33 @@ env:
     - "TEST_SUITE=unittests"
     - "TEST_SUITE=integrationtests"
     - "TEST_SUITE=integrationtestsdev"
+
+dist: xenial
+
+sudo: required
+
```

Comment:
Python 3.7 requires xenial for security patches. And sudo must also be set for it to work.